### PR TITLE
NTP improvements, fixes F23

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: pkg/systemd/host.js [hunk 1]
```diff
@@ -148,7 +148,8 @@ function ServerTime() {
         client.call(timedate.path,
                     "org.freedesktop.DBus.Properties", "Get", [ "org.freedesktop.timedate1", "NTPSynchronized" ]).
             fail(function (error) {
-                if (error.name != "org.freedesktop.DBus.Error.UnknownProperty")
+                if (error.name != "org.freedesktop.DBus.Error.UnknownProperty" &&
+                    error.message != "not-found")
```
Comment (stefwalter) [inline pkg/systemd/host.js]: I think using ```error.problem``` would be better here. The message can change depending on who comes around and edits ```cockpit.message()```

File: pkg/systemd/host.js [hunk 2]
```diff
@@ -228,15 +229,19 @@ PageServer.prototype = {
             $elt.show();
 
             var model = {
-                Synched: self.server_time.timedate.NTPSynchronized
+                Synched: self.server_time.timedate.NTPSynchronized,
+                service: "ntpd.service"
```
Comment (stefwalter) [inline pkg/systemd/host.js]: Why is there an assumption that ntpd.service will be in use when timesyncd is not? What about chronyd? Shouldn't we be treating all non timesyncd as one case?

**Comment by stefwalter** [comment]
This also should remove the known issue in ```test/naughty/```

File: pkg/systemd/host.js (hunk 1 above)
Comment (mvollmer) [inline pkg/systemd/host.js]: Yes, very good.

File: pkg/systemd/host.js (hunk 2 above)
Comment (mvollmer) [inline pkg/systemd/host.js]: Yes, I agree.

**Comment by mvollmer** [comment]
> This also should remove the known issue in test/naughty/

Done.

File: pkg/systemd/host.js [hunk 3]
```diff
@@ -148,7 +148,9 @@ function ServerTime() {
         client.call(timedate.path,
                     "org.freedesktop.DBus.Properties", "Get", [ "org.freedesktop.timedate1", "NTPSynchronized" ]).
             fail(function (error) {
-                if (error.name != "org.freedesktop.DBus.Error.UnknownProperty")
+                console.log(error);
```
Comment (stefwalter) [inline pkg/systemd/host.js]: Below the messages are quietened down, and yet this logs everything. Is that intentional?

Comment (mvollmer) [inline pkg/systemd/host.js]: Damn, debug leftovers.

**Pull request closed without merging**